pyopenjtalk/htsengine.pyx

```diff
@@ -19,7 +20,7 @@ from .htsengine cimport (
     HTS_Engine_get_generated_speech, HTS_Engine_get_nsamples
 )
 
-cdef class HTSEngine(object):
+cdef class HTSEngine:
```

Comment:
This can be made unnecessary by setting `language_level=3`.

https://cython.readthedocs.io/en/3.0.x/src/changes.html#id33
>- With language_level=3/3str, Python classes without explicit base class are now new-style (type) classes also in Py2. Previously, they were created as old-style (non-type) classes. (Github issue [#3530](https://github.com/cython/cython/issues/3530))